test/js_interop_helpers_test/js_function_test_prod.html

```diff
@@ -5,7 +5,7 @@
     <title></title>
 
     <script src="packages/react/react_prod.js"></script>
-    <script src="packages/react/react_dom_prod.js"></script>
+    <script src="packages/react/react_dom.js"></script>
```

Comment:
Oh, I just changed all `react_dom_prod.js` to `react_dom.js`, this one can be reverted